WordPress/Classes/Models/Blog+Editor.swift

```diff
@@ -29,6 +29,9 @@ extension Blog {
     }
 
     @objc var isGutenbergEnabled: Bool {
-        return self.editor.mobile == .gutenberg
+        guard let selectedEditor = editor.mobile else {
+            return GutenbergSettings().getDefaultEditor(for: self) == .gutenberg
```

Comment:
This felt like it should go inside the `Editor` initialization, but then the whole `Editor` struct felt a bit superfluous so I'd like to propose a different API for the blog editor settings: 5a2600e8cc8ef0983fd3e4f91dcf9704424c6508

Reply:
Absolutely! Looks a lot simpler and more elegant.
You can make a small PR and I'll ✅ right away?

Thanks for the suggestion 🙏

Reply:
Cherry-picked the commit for easier integration with this PR